### ISSUE TITLE
require image load wrong image in android

### ISSUE THREAD

**Issue by invyctus92** [facebook/react-native#32671]
### Description

Whenever I add a new image to the project, the next update presents this problem in production, both in apk and aab. The problem is that some of the images in the app point to other images and not to those in source. for example instead of a wave or icon, I get something else. This does not happen in the android emulator, it only happens if I update the app from a previous version, if I install it from scratch it does not access. I uploaded two photos from two different phones, one before the update, one with the update.  Can you give me a hand? 

### Version

0.66.3

### Output of `react-native info`

    OS: macOS 10.15.7
    CPU: (4) x64 Intel(R) Core(TM) i5-3330S CPU @ 2.70GHz
    Memory: 52.42 MB / 8.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.22.1 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.14.12 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.10.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2
      System Images: android-30 | Google APIs Intel x86 Atom, android-30 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2020.3 AI-203.7717.56.2031.7784292
    Xcode: 12.4/12D4e - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.13 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.66.3 => 0.66.3 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

Whenever I add a new image to the project

### Snack, code example, screenshot, or link to a repository

![1638191929753](https://user-images.githubusercontent.com/13299821/143875383-b4b1a283-0b32-4f7b-aca4-6d300b338e3f.jpg)
![Screenshot_20211129_142013_com muvsrl](https://user-images.githubusercontent.com/13299821/143875396-f40775b6-011e-4d39-8a91-c97cc34e4333.jpg)

**Comment by invyctus92** [comment]
Solved, problem was affecting Fresco, in particular version 2.6.0, downgrade to 2.5.0 I solved it

**Comment by dimaportenko** [comment]
For those who don't know how to do it. You can go to the `android` dir and run 
```
./gradlew app:dependencies
```
it will show your android dependecies. 

In the `android/app/build.gradle` add following

```
configurations.all {
    resolutionStrategy {
        force "com.facebook.fresco:fresco:2.5.0"
    }
}
```
I placed it at the same level as the dependencies section.

**Comment by dimaportenko** [comment]
@pierroo 
> I have no mention of fresco in my app/build.gradle

It is most likely 3d party dependency.  Try `./gradlew app:dependencies` in the android directory to see the android dependencies tree. 

> where I should place it in the file?

check my previous comment 

> I placed it at the same level as the dependencies section.

```
...
configurations.all {
    resolutionStrategy {
        force "com.facebook.fresco:fresco:2.5.0"
    }
}

dependencies {
    implementation fileTree(dir: "libs", include: ["*.jar"])
    //noinspection GradleDynamicVersion
    implementation "com.facebook.react:react-native:+"  // From node_modules
...
```

**Comment by pierroo** [comment]
Thank you @dimaportenko 

So according to your comment, the configurations.all should come BEFORE dependencies, so I just did then ran gradlew app:dependencies , which gave me:

![image](https://user-images.githubusercontent.com/17619412/197332689-57beccbb-b069-4274-b25b-eb677e172885.png)

(I extracted the part that mentioned fresco)
So here we can see fresco is at 2.5.0, but then for some reason below it goes 2.5.0 -> 2.6.0
=> is this something I should worry about?
Should I add lines in resolutionStrategy to force 2.5.0 on all those lines that mention 2.6.0?

Again, thank you so much for your help, this is one mean bug right here that is completely messing my app in production. :/
